web/packages/teleport/src/components/TdpClientCanvas/TdpClientCanvas.tsx

```diff
@@ -401,6 +438,7 @@ export type Props = {
   canvasOnMouseWheelScroll?: (cli: TdpClient, e: WheelEvent) => void;
   canvasOnContextMenu?: () => boolean;
   style?: CSSProperties;
+  updatePointer: boolean;
```

Comment:
I think we can make this property optional, you won't have to pass `false` to `DesktopPlayer`.
```suggestion
  updatePointer?: boolean;
```